docs/content_management/block_reference.md

```diff
@@ -36,167 +36,206 @@ The following blocks are provided with a clean installation of [[= product_name
 
 [[= include_file('docs/content_management/create_edit_pages.md', 86, 96) =]]
 
+## Block basics
+
+Handling blocks is straightforward, drag and drop them into the desired spot within the drop zone. 
+There's no need to worry about getting their placement perfect initially, you can rearrange them whenever you need.
+
+### Settings
+
+You have the ability to control individual blocks by accessing their settings. 
+Click on the settings icon next to the block's name. 
+
+![Block settings](img/block_settings.png "Block settings")
+
+The settings include:
+
+- Move Up — change the block's position by shifting it upwards.
+- Move Down — change the block's position by shifting it downwards.
+- Configuration — access the configuration window.
+- Duplicate — create a copy of the block below the original, including its settings.
+- Refresh — update the block's preview.
+- Delete — remove the existing block.
+
+### Design tab
+
+In the block configuration on the **Design** tab, under the **View** field, you can select the layout for presenting a block.
+
+### Scheduler tab
+
+In the block configuration on the **Scheduler** tab, you can set a date for the block to be revealed and hidden.
+
 ## Banner block
 
-On the **Basic** tab, set values in the following fields:
+The banner block showcases an image linked to a specific URL.
+It can be used for creating visually appealing banners or promotional sections on websites that require linked images to direct users to specific pages or content.
+On the **Properties** tab, set values in the following fields:
 
-- **Name** - Enter a name for the Page block.
-- **Image** - Click **Select content**, navigate through the content and select an image to display.
-- **URL** - Enter a URL to open when clicking the selected image.
+- **Name** — Enter a name for the page block.
+- **Image** — Click **Select content**, browse the available content, and choose an image for display.
+- **URL** — Enter a URL that opens when users click on the banner block.
 
 ## Bestsellers block
 
-On the **Basic** tab, set values in the following fields:
+Displays products that have recently been top sellers, helping users discover popular items quickly.
```

Comment:
```suggestion
Displays products that were recently sellers, to help users discover popular items quickly.
```

While it is not wrong, simplified English suggests avoiding perfect tenses and using simple tenses.

Reply:
```suggestion
Displays products that were recently top sellers, to help users discover popular items quickly.
```